grails-app/controllers/org/pih/warehouse/UrlMappings.groovy

```diff
@@ -490,6 +490,12 @@ class UrlMappings {
             action = [GET: "invoiceTypeCodes"]
         }
 
+        // TODO: Investigate the proper way to handle validation as a REST resource
+        "/api/invoiceItems/$id/validation"(parseRequest: true) {
```

Comment:
AFAIK parseRequest is no longer applicable in Grails 3.

https://pihemr.atlassian.net/browse/OBGM-401?focusedCommentId=139786
https://stackoverflow.com/a/24048120/136597